components/toast/index.css

```diff
@@ -100,27 +92,20 @@
 
 .spectrum-Toast--info {
 	background-color: var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default));
-	color: var(--mod-toast-informative-background-color-default, var(--spectrum-toast-informative-background-color-default));
-
-	.spectrum-Toast-closeButton:focus-visible:not(:active) {
```

Comment:
Oh! I see one more of these in the `--negative` block. We can remove that as well, from what I can tell.